### PR TITLE
Calculator Events

### PR DESCRIPTION
**Pull request by waqaskanju**
In this pull events are added to calculator to do its operation.
- calculate.js file is added
- Operate.js file is added
- big.js library is added through npm
- events are added to buttons

### CODE REVIEW

File: src/components/calc2.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
+// Calculator.js
+/* eslint-disable */
+import React from 'react';
+import calculate from '../logic/calculate';
+import operate from '../logic/operate';
+import Big from 'big.js';
+/* import './Calculator.css'; */
```
Comment (ahmetbozaci) [inline src/components/calc2.js]: - [ ] Kindly delete unused commented code to make clean code.

File: src/components/calc2.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
+// Calculator.js
+/* eslint-disable */
```
Comment (ahmetbozaci) [inline src/components/calc2.js]: - [ ] Please don't disable eslint. If it is giving an error kindly fix it.

File: src/components/calc2.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+          <button onClick={this.btnClick} value="0" class="key key0">0</button>
+          <button onClick={this.btnClick} value="." class="key">.</button>
+          <button onClick={this.btnClick} value="=" class="key c4">=</button>
+        </div>
+      </div>
+    );
+  }
+}
+
+export default Calc;
```
Comment (ahmetbozaci) [inline src/components/calc2.js]: - [ ] When ı click the symbols it is giving showing undefined. Kindly fix it.
- [ ] +/- and * operators are not working. Please fix it.

https://user-images.githubusercontent.com/38635158/176957907-05b5004e-c9fd-4901-b064-ab3564308502.mp4

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️


 @waqaskanju  ✋🏼 

#### Your project is excellent. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Nice to have 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- You have one commit for everything. For best practices, you should commit often. [Git Commit Best Practices](https://www.perforce.com/blog/vcs/git-best-practices-git-commit)
- Your commit messages should always start with capital letters and they should be an imperative tense. I see some of them are not like that. Instead of writing `Added style or Created the file` you should write `Add style or Create the file`
- ***I write it as optional. But this is a [general requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#:~:text=The%20commit%20message%20should%20be%20written%20in%20the%20imperative%20tense%20and%20capitalized) for every project. So you should pay attention to this in your next projects.***

**OPTIONAL**

- You can change them by using `git-rebase`. If you want to know how to change ı leave a few resources.
- [Video - How to change commit messages](https://www.loom.com/share/00771f369b7b47fc968c019ecf0d3ba6)
- [Article - How to create good commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/)
___

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

File: src/components/calc2.js (hunk 3 above)
Comment (waqaskanju) [reply src/components/calc2.js]: thanks for showing the errors.
I have fixed it. Thanks

**Review by hiromataba — APPROVED** [review]
# Status: Approved :tada: :tada: :tada: 
Hi @waqaskanju     :wave: :wave: 

200w  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**